lib/media/streaming_engine.js

```diff
@@ -1562,7 +1562,7 @@ shaka.media.StreamingEngine = class {
 
     if (mediaState.segmentPrefetch && mediaState.segmentIterator &&
         !this.audioPrefetchMap_.has(mediaState.stream)) {
-      mediaState.segmentPrefetch.evict(reference.startTime);
+      mediaState.segmentPrefetch.evict(reference.startTime + 0.001);
```

Comment:
Please hide it behind a constant and/or add a comment so it's obvious why we need that.